### PR TITLE
Add nightstand news item

### PR DESCRIPTION
**Pull request by beroset**
This is a new news article that describes a little of what Nightstand mode is and how it came about.

### CODE REVIEW

**Comment by FlorentRevest** [comment]
Similar torture as for @eLtMosen https://github.com/AsteroidOS/asteroidos.org/pull/260#issuecomment-1459068986 Could you convert this article to Markdown so future news writers don't get too intimidated ?

File: pages/news/nightstand.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+posted: 2023-03-08
+post: true
+thumbnails: nightstand
+summary: The new Nighstand mode shows the state of a charging watch at a glance.
+author: Ed Beroset (beroset)
+---
+
+<p>When a watch is on the charger, it's useful to be able to see the state of charge at a glance.  We've implemented a new feature called *Nightstand mode* that helps with this.</p>
+
+<p>As is often the case with AsteroidOS development, there is often conversation about what new features should be implemented and how they should work.  To help explain to the rest of the community what Nightstand mode might do, and how it might work, the following scenario and requirements descriptions were inserted into a <a href="https://github.com/AsteroidOS/asteroid/issues/151">issue description</a>.</p>
```
Comment (FlorentRevest) [inline pages/news/nightstand.md:13]: To keep this an easy read we can probably simplify this quite a bit by just saying something along the line of:

The feature was originally requested in a GitHub issue (link to the specific comment https://github.com/AsteroidOS/asteroid/issues/151#issuecomment-1204035843) and described in these terms:

File: pages/news/nightstand.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+<h2>Scenario</h2>
+
+<p>A tired traveler arrives at a hotel late at night and finds there is no clock in the room.  Fortunately, this person has an AsteroidOS watch with nightstand mode.  She uses the settings to select the kind of display that she would like to use for this and enables the mode.  Then she puts the watch on its charging stand where it acts as sort of a bedside clock, prominently featuring the state of charge and the current time.  She wears glasses and does not want to have to put them on to see the time, so she chooses a watchface that is high contrast and features large numbers for the time.  Even with that, it's not always that easy to see the time, so it is configured so that at 3AM with no glasses, she can pick up the watch and bring it closer to note the time (ugh!  3AM!) and replace it on the stand within some configurable period of time with the watch not leaving nightstand mode.
```
Comment (FlorentRevest) [inline pages/news/nightstand.md:17]: Keep the scenario which is entertaining to read. Maybe you could quote it in the style of eltmoset's article on gadgetbridge ?

Also, it looks like it's missing a `</p>` ? probably it'll be easier once written in Markdown :)

File: pages/news/nightstand.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+<h2>Requirements</h2>
```
Comment (FlorentRevest) [inline pages/news/nightstand.md:19]: But skip the whole requirements part which is too abstract for an announcement and contains a lot of repetitions of the above paragraph.

File: pages/news/nightstand.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+</ol>
+
+<h2>Implementation</h2>
+<p>All of the requirements except rotation based on orientation are now implemented.</p>
+<p>After some discussion among developers, the way it works now is that if Nightstand mode is active and the watch is placed on a charger, by default, the currently selected watchface will be used for this mode.  Typically, this means that each watchface has sort of an alternative mode that adds some kind of charge indicator ring.</p>
+<p>To accomodate our glasses-wearing user as described in the scenario above, one can also select a custom watchface that will be used just when Nightstand mode is active.  This allows users to choose a complex and information-filled watchface during the day, but an easier-to-read version for nightstand mode.</p>
+<p>This mode is most useful when paired with Always-on display.</p>
+
+<h2>Your ideas are welcome!</h2>
+<p>Many of the features that are now implemented in AsteroidOS come from suggestions from users.  If you have ideas for a new feature or comments or questions on this one, drop by <a href="https://matrix.to/#/#Asteroid:matrix.org">Matrix chat</a> and say hello.</p>
```
Comment (FlorentRevest) [inline pages/news/nightstand.md:39]: Would "drop by Matrix" or "drop by the Matrix chat" work better than "drop by Matrix chat" ? idk you're the native speaker :)

File: pages/news/nightstand.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+<li>The user should be able to disconnect the charger for a brief period without changing from the nightstand display</li>
+<li>After the brief period expires, the display should revert to the regular watchface</li>
+<li>The duration of the brief period should be configurable from 0 to, say, 30 seconds</li>
+<li>When nightstand mode is on, the display should rotate based on orientation</li>
+</ol>
+
+<h2>Implementation</h2>
+<p>All of the requirements except rotation based on orientation are now implemented.</p>
+<p>After some discussion among developers, the way it works now is that if Nightstand mode is active and the watch is placed on a charger, by default, the currently selected watchface will be used for this mode.  Typically, this means that each watchface has sort of an alternative mode that adds some kind of charge indicator ring.</p>
+<p>To accomodate our glasses-wearing user as described in the scenario above, one can also select a custom watchface that will be used just when Nightstand mode is active.  This allows users to choose a complex and information-filled watchface during the day, but an easier-to-read version for nightstand mode.</p>
```
Comment (FlorentRevest) [inline pages/news/nightstand.md:35]: I have the feeling that these few paragraphs can be simplified a lot to be an easier read for end-users. Even I who reviewed the code had to read them slowly and a couple of times to get the idea.

Something in the ~~rough direction of:

This feature is now fully-implemented! All watchfaces have been extended with a charge indicator ring that fits their style and which only shows up in night stand mode. By default, the same watchface that's used in day-time is also used for nightstand mode but to accomodate our glasses-wearing user, one can also select a custom watchface only used in Nighstand mode.

To support rotated chargers, we also have plans to extend Nighstand with auto-rotate in the future.

Don't feel like you have to re-use my text verbatim, it's just an example that keeps most of your ideas but, I feel, said in simpler terms ?

-----

Also if it turns out to be a very simple article in the end, the h2 titles might appear to be unnecessary ? Not sure about this yet, we'll see how that turns out

**Review by FlorentRevest — COMMENTED** [review]
This will definitely be a great article to have thanks a lot for writing it! (and the feature too! :) )

Let's schedule it for a week or so after https://github.com/AsteroidOS/asteroidos.org/pull/260

File: pages/news/nightstand.md (hunk 2 above)
Comment (beroset) [reply pages/news/nightstand.md:17]: I started in markdown, but our news articles do not support it at the moment and I didn't take the time to dig into that yet.  Also, I started by using `<blockquote>` but that caused the text to be light gray and didn't look good.  I didn't have tome to dive into the templates at the moment.

File: pages/news/nightstand.md (hunk 3 above)
Comment (beroset) [reply pages/news/nightstand.md:19]: You're probably right.  I was trying to show that the the scenario leads to the requirements and that the implementation matches the requirements, but we can omit it.

File: pages/news/nightstand.md (hunk 4 above)
Comment (beroset) [reply pages/news/nightstand.md:39]: Any of the three works grammatically in English.  I didn't use the first because people not already familiar with Matrix might not know what it is.

File: pages/news/nightstand.md (hunk 5 above)
Comment (beroset) [reply pages/news/nightstand.md:35]: These are good suggestions.  I'll simplify and resubmit.  Thanks!

File: pages/news/nightstand.md (hunk 2 above)
Comment (FlorentRevest) [reply pages/news/nightstand.md:17]: Ugh. I see :( then forget about Markdown for now ...

Comment (beroset) [reply pages/news/nightstand.md:17]: Fixed!  Merge #280 first and we'll be able to use this updated markdown version.

**Comment by beroset** [comment]
Note that this now required #280 to be merged first.

File: pages/news/nightstand.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+---
+title: Nightstand mode
+layout: news
+posted: 2023-03-28
+post: true
+thumbnails: nightstand
+summary: The new Nighstand mode shows the state of a charging watch at a glance.
+author: Ed Beroset (beroset)
+---
+When a watch is on the charger, it's useful to be able to see the state of charge at a glance.  We've implemented a new feature called **Nightstand mode** that helps with this.  It was originally requested in an [this issue](https://github.com/AsteroidOS/asteroid/issues/151) and described like this:
```
Comment (FlorentRevest) [inline pages/news/nightstand.md:10]: In this issue* (no an)

File: pages/news/nightstand.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+# Features #
+By default, when the Nightstand mode is enabled and the charger is connected, the watch changes to a nightstand version of the current watchface which typically adds a green charge indicator ring around the watchface.
+
+To accomodate our glasses-wearing user as described in the scenario above, one can also select a custom watchface that will be used just when Nightstand mode is active.  This allows users to choose a complex and information-filled watchface during the day, but an easier-to-read version for nightstand mode.
+
+Finally, when removed from the charger and after a configurable Delay time (from 0 to 30 seconds), the watch reverts to "daytime mode."
```
Comment (FlorentRevest) [inline pages/news/nightstand.md:21]: No capital d for delay ?

And maybe just delay instead of delay time ? Not sure about this though

File: pages/news/nightstand.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+This mode is most useful when paired with Always-on display.
```
Comment (FlorentRevest) [inline pages/news/nightstand.md:23]: Maybe just to make it clearer I'd complete this with "so nightstand mode can be seen without having to unlock the watch" or something like that

**Review by FlorentRevest — APPROVED** [review]
Looks great! 😃 I just had a few nits but it's taking good shape, thanks a lot!

File: pages/news/nightstand.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+# Your ideas are welcome! #
+Many of the features that are now implemented in AsteroidOS come from suggestions from users.  If you have ideas for a new feature or comments or questions on this one, drop by <a href="https://matrix.to/#/#Asteroid:matrix.org">Matrix chat</a> and say hello.
```
Comment (eLtMosen) [inline pages/news/nightstand.md:26]: hehe, i am confused here.
Is this `a href` supposed to be changed to markdown still?

**Review by eLtMosen — COMMENTED** [review]
Very cool overall. Just that one markdown change left afaics.

File: pages/news/nightstand.md (hunk 6 above)
Comment (beroset) [reply pages/news/nightstand.md:10]: Good catch!

File: pages/news/nightstand.md (hunk 7 above)
Comment (beroset) [reply pages/news/nightstand.md:21]: I originally intended to echo the exact setting name, but you're right on both.  Thanks!

File: pages/news/nightstand.md (hunk 8 above)
Comment (beroset) [reply pages/news/nightstand.md:23]: Good point.

File: pages/news/nightstand.md (hunk 9 above)
Comment (beroset) [reply pages/news/nightstand.md:26]: Good catch!

**Comment by beroset** [comment]
Ooops!  Sorry for the error.  This should fix it.

**Comment by FlorentRevest** [comment]
Looks good! Thank you :D as said above, let's merge it in a few days. Remind me if I forget.

**Comment by FlorentRevest** [comment]
Thank you! 🎉

I realized that we should also change dates if we want to play this sort of news release strategy game but... just having news is already much better than not having any ahah.
